.github/workflows/release.yml

```
@@ -63,6 +63,10 @@ jobs:
         DOCKER_USERNAME: ${{ env.DOCKER_USERNAME }}
         DOCKER_PASSWORD: ${{ env.DOCKER_PASSWORD }}
     
+    - name: Pacakge windows images
```

Spelling nit

```suggestion
    - name: Package windows images
```

Also, shouldn't this go above, either directly below the existing `Package Images` step, or perhaps just even another `run` line in that same step?

---

I will fix the typo, however the order is actually what it used to be in drone
https://github.com/rancher/rke2/blob/v1.28.11-rc6%2Brke2r1/.drone.yml#L160C1-L172C1

---

ok!

We need to review the makefile when we get rid of dapper, I think we are rebuilding the same stuff many times.